staging/src/k8s.io/apiserver/pkg/server/egressselector/config_test.go

```diff
@@ -108,7 +108,7 @@ egressSelections:
 						},
 					},
 					{
-						Name: "master",
```

Comment:
As long as master is a valid (but deprecated) option we should have a test for it.

Reply:
Good catch, added a new unit test for it